### DISCUSSION TITLE
Calendar week when changing user ?

### DISCUSSION THREAD

**Discussion by randomev** [kimai/kimai#3949] [General]
Hello

I there something I could do to keep week from changing back to current week when changing user from calendar dropdown ?

Use case is that every week I do last week invoices and first I quickly look through what each employee has done. And I like to do this with calendar view as this gives a really nice quick overview. So I scroll back to week in interest in calendar view. But when I change employee from top right dropdown, week is reset to current week. Then I need to scroll to week in interest always again. This happens every time and I'm looking if there is anything I could do myself or can this be marked as a feature request for next versions.

We are using paid subscription in the Kimai cloud.

Best regards,

Henkka

**Comment by kevinpapst** [comment]
See feature request at: https://github.com/kimai/kimai/issues/3952

Closing here.
